app-template/src/AppInitializer.tsx

```diff
@@ -49,7 +53,45 @@ class AppInitializer extends React.Component<AppInitializerProps, AppInitializer
       </TokenContext.Provider>
     )
   }
+
+  /**
+   * PORTALS-490: Set Synapse callback cookie
+   * Will attempt to set a .synapse.org domain cookie that has enough information to lead the user
+   * back to this portal after visiting www.synapse.org.
+   */
+  updateSynapseCallbackCookie() {
+    let color = 'white'
+    let background = '#4db7ad'
+    let name = ''
+    let icon = ''
+    const footerElement = document.querySelector('#footer')
+    if (footerElement) {
+      color = window.getComputedStyle(footerElement, null).getPropertyValue('color')
+      background = window.getComputedStyle(footerElement, null).getPropertyValue('background-color')
+    }
+    const homeLinkImgElement = document.querySelector('#home-link img')
+    if (homeLinkImgElement) {
```

Comment:
a note to myself -- I need to change this to use the footer icon once it gets added